examples_tests/42.FragmentShaderPathTracer/main.cpp

```diff
@@ -114,6 +115,24 @@ int main()
 
 	nbl::video::IGPUObjectFromAssetConverter CPU2GPU;
 	
+	// For CPU2GPU Params
+	auto pool_transfer = device->createCommandPool(transferUpQueue->getFamilyIndex(), IGPUCommandPool::ECF_RESET_COMMAND_BUFFER_BIT);
+	core::smart_refctd_ptr<IGPUCommandPool> pool_compute;
+
+	if(transferUpQueue->getFamilyIndex() == computeQueue->getFamilyIndex())
+		pool_compute = pool_transfer;
+	else
+		pool_compute = device->createCommandPool(computeQueue->getFamilyIndex(), IGPUCommandPool::ECF_RESET_COMMAND_BUFFER_BIT);
+	
+	core::smart_refctd_ptr<IGPUCommandBuffer> transferCmdBuffer;
+	core::smart_refctd_ptr<IGPUCommandBuffer> computeCmdBuffer;
+
+	device->createCommandBuffers(pool_transfer.get(), IGPUCommandBuffer::EL_PRIMARY, 1u, &transferCmdBuffer);
+	device->createCommandBuffers(pool_compute.get(), IGPUCommandBuffer::EL_PRIMARY, 1u, &computeCmdBuffer);
+	
+	cpu2gpuParams.perQueue[IGPUObjectFromAssetConverter::EQU_TRANSFER].cmdbuf = transferCmdBuffer;
+	cpu2gpuParams.perQueue[IGPUObjectFromAssetConverter::EQU_COMPUTE].cmdbuf = computeCmdBuffer;
```

Comment:
CommonAPI already fills a lot of `cpu2gpuParams` for us, any chance of moving this there?

Reply:
Ok then :)